experimental/dds/tree2/src/test/class-tree/schemaFactory.spec.ts

```diff
@@ -262,7 +262,7 @@ describe("schemaFactory", () => {
 		const view: TreeView<Canvas> = tree.schematize(config);
 		const stuff = view.root.stuff;
 		assert(stuff instanceof NodeList);
-		const item = stuff[1];
+		const item = stuff[0];
```

Comment:
Unrelated bug fix?

Reply:
Yea - you can see the list being initialized up above with only one thing in it. The test itself is not unrelated though - it was skipped because it needed the functionality that is added in this PR.